manifests/pyvenv.pp

```diff
@@ -48,7 +48,7 @@
       $python3_venv_package = "python${normalized_python_version}-venv"
 
       case $facts['os']['distro']['codename'] {
-        'xenial', 'bionic', 'cosmic', 'disco', 'stretch', 'buster': {
+        'xenial', 'bionic', 'cosmic', 'disco', 'stretch', 'buster', 'focal': {
```

Comment:
that's all the ubuntu/debian versions we support. I think we can just drop this case statement?

Reply:
Ya, I dropped it.